### ISSUE TITLE
Can't configure Backblaze after initial install

### ISSUE THREAD

**Issue by joeladria** [JonathanTreffler/backblaze-personal-wine-container#41]
<img width="907" alt="image" src="https://user-images.githubusercontent.com/2769376/215278853-73332c88-13bb-4f4a-822e-0e668ba2d04c.png">

Did a clean install, and once complete I can't click anything in the settings menu that pops up. I can restart the docker, but after that I just get this, and no other UI responds either. It does seem to be backing up to my account, but can't configure any additional drives or settings. I can maybe hack away at the XML files in the config files, but seems a little sketchy.

 Any ideas?

**Comment by monteroman** [comment]
I'm seeing this on an existing installation after upgrading. It seems that this latest version of v8.0.1 is freezing up in the GUI.

**Comment by mxmmxx** [comment]
I had this issue too, using an older executable version from [archive.org](https://web.archive.org/web/20230000000000*/https://secure.backblaze.com/win32/install_backblaze.exe) solved it for me

**Comment by Lesani** [comment]
have the same issue, first I was on the docker container from unraid, then I tried step-for-step the wiki instructions. of course I removed the config directory beforehand.

I noticed that according to the wiki it automatically should install and prompt for Wine Gecko, however in the actual install, there is no step for Gecko. An old version (v7.x) worked, but does not properly backup for some reason.

**Comment by monteroman** [comment]
Lesani, if you are using Unraid, edit the container and add :main to it and let it update. 
Try your setup again. 
![image](https://user-images.githubusercontent.com/5451042/216579330-20fef524-816c-4535-adb0-98a8b1f19659.png)

I've been having really good luck with the latest version of Wine and the latest version of BackBlaze.

**Comment by frod123** [comment]
I am using Unraid and had this issue. Using the suggestion from @monteroman fixed all my issues 100%.

**Comment by timothe** [comment]
I can't click on anything but I'm on Synology. Is there an ubuntu18 version of the main branch? 
Currently the ubuntu18 branch doesn't allow me to click on settings and add the D drive (but wine sees it).
The main branch is giving me the "could not communicate" error message.

**Comment by timothe** [comment]
Side note: I see a new build from main on Dockerhub, but on this repo both the main and ubuntu18 Dockerfiles are from a month ago. Is the code here up to date?

**Comment by timothe** [comment]
@monteroman are you on Synology?

**Comment by monteroman** [comment]
I am not. I am using Unraid.

**Comment by datbird** [comment]
> Lesani, if you are using Unraid, edit the container and add :main to it and let it update. Try your setup again. ![image](https://user-images.githubusercontent.com/5451042/216579330-20fef524-816c-4535-adb0-98a8b1f19659.png)
> 
> I've been having really good luck with the latest version of Wine and the latest version of BackBlaze.

This right here, this is the way. Solved all my problems!

**Comment by JonathanTreffler** [comment]
The :main workaround should no longer be necessary, I just released version 1.4, so :latest now contains the same dependency versions as :main.
Nice work on the community collaboration, there are quite a few issues in this repository so I don't have enough time to reply to them all, so thanks to everyone sharing their solutions.

> Is there an ubuntu18 version of the main branch?

currently not, but :ubuntu18 got updated with 1.4 too, so maybe the issue got fixed for ubuntu18 too.
